### PR TITLE
Added a new line to the README.md file

### PR DESCRIPTION
**Pull request by VeronikaDmytryk**

### CODE REVIEW

**Comment by opensso** [comment]
@VeronikaDmytryk , test message. Please reply!

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-Here is some changes
\ No newline at end of file
+Here is some changes
+Added new line to the file for new branch
```
Comment (opensso) [inline README.md]: I think this is great. Please reply

Comment (opensso) [reply README.md]: Did you know you can use `markdown` **here** ?

Comment (VeronikaDmytryk) [reply README.md]: Replying

Comment (VeronikaDmytryk) [reply README.md]: I could. Thank you for an idea

**Pull request merged**